stdlib/src/builtin/string.mojo

```diff
@@ -438,6 +438,17 @@ struct String(
     var _buffer: Self._buffer_type
     """The underlying storage for the string."""
 
+    """ Usefull string aliases. """
+    alias WHITESPACE = String(" \n\t\r\f\v")
+    alias ASCII_LOWERCASE = String("abcdefghijklmnopqrstuvwxyz")
+    alias ASCII_UPPERCASE = String("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+    alias ASCII_LETTERS = String.ASCII_LOWERCASE + String.ASCII_UPPERCASE
+    alias DIGITS = String("0123456789")
+    alias HEXDIGITS = String.DIGITS + String("abcdef") + String("ABCDEF")
```

Comment:
If we would not follow Python, this can be renamed to: HEX_DIGITS and OCTDIGITS -> OCT_DIGITS

Reply:
Yep, looks good to me, because we're already breaking python naming here, we can change this as well.